core/src/main/scala/org/apache/spark/ui/JWSFilter.scala

```diff
@@ -52,7 +52,7 @@ private class JWSFilter extends Filter {
    * - WeakKeyException will happen if the user-provided value is insufficient
    */
   override def init(config: FilterConfig): Unit = {
-    key = Keys.hmacShaKeyFor(Decoders.BASE64URL.decode(config.getInitParameter("key")));
+    key = Keys.hmacShaKeyFor(Decoders.BASE64URL.decode(config.getInitParameter("secretKey")));
```

Comment:
Currently, `key` can be exposed if `spark.ui.filters` configuration is removed mistakenly.
![Screenshot 2024-08-03 at 22 02 55](https://github.com/user-attachments/assets/1e8c8f19-5cf5-4696-8ad3-fd86c001d052)

Reply:
As you make the change https://github.com/apache/spark/pull/47596#discussion_r1703039544, do we still need to change this param to `secretKey`?